tests/test_client.py

```diff
@@ -83,6 +83,12 @@ def test_index_list_with_params(client, user):
     assert data_by_ids.status_code == 200
     data_list_all = data_by_ids.json
```

Comment:
This isn't being tested anymore since in ln 90 `data_list_all` replaces this one.

Reply:
So i  guess i would suggest add the same types of checks in line 92 over here.